### ISSUE TITLE
Please keep the original credits for my work

### ISSUE THREAD

**Issue by shaked6540** [dardizzola/YouTubeDownloaderAllInOne#6]
I see that you modified the program a little, I'm ok with that as long as you don't remove any credit for me or the translators, specifically in https://github.com/dardizzola/YouTubeDownloaderAllInOne/blob/main/LaMuccaRossaVideoDownloader/Languages/English.xaml#L109

Thanks

**Comment by dardizzola** [comment]
You are right.I’ll do it.Inviato da mobileIl giorno 13 lug 2024, alle ore 16:23, Shaked Dahan ***@***.***> ha scritto:﻿
I see that you modified the program a little, I'm ok with that as long as you don't remove any credit for me or the translators, specifically in https://github.com/dardizzola/YouTubeDownloaderAllInOne/blob/main/LaMuccaRossaVideoDownloader/Languages/English.xaml#L109
Thanks
